[3.1.4] compilation error on systems having inclusion paths for nss different from expected ones

**Issue Description**
Compilation fails on systems where `nss` inclusion path is different from the expected one (`nss3`), e.g. In ALT it's `/usr/include/nss`.

**Package Version and Platform:**
 - Platform: ALTLinux
 - Package and version: 389-ds-base 3.1.4

**Actual result**
```
    ldap/servers/slapd/dyncerts.c:21:10: fatal error: nss3/pk11pub.h: No such file or directory
       21 | #include <nss3/pk11pub.h>
          |          ^~~~~~~~~~~~~~~~
    compilation terminated.
    make[1]: *** [Makefile:9311: ldap/servers/slapd/libslapd_la-dyncerts.lo] Error 1
    make[1]: Leaving directory '/usr/src/RPM/BUILD/389-ds-base-3.1.4'
```

See also:
https://github.com/389ds/389-ds-base/issues/3411

--- a/ldap/servers/slapd/dyncerts.c
+++ b/ldap/servers/slapd/dyncerts.c
@@ -18,8 +18,8 @@
 #include "slap.h"
 #include "fe.h"
 #include <nss.h>
-#include <nss3/pk11pub.h>
-#include <nss3/certdb.h>
+#include <pk11pub.h>
+#include <certdb.h>
 #include "svrcore.h"
 #include "dyncerts.h"
 

